server/schemas/meteorological.py

```diff
@@ -29,23 +26,23 @@ class Historical_Meteorological_Base(BaseModel):
 
 
 # Properties to receive on item creation
-class Historical_Meteorological_Create(Historical_Meteorological_Base):
+class MeteorologicalCreate(MeteorologicalBase):
     ...
 
 
 # Properties to receive on item update
-class Historical_Meteorological_Update(Historical_Meteorological_Base):
+class MeteorologicalUpdate(MeteorologicalBase):
     ...
 
 
 # Properties shared by models stored in DB
-class Historical_Meteorological_InDBBase(Historical_Meteorological_Base):
+class MeteorologicalInDBBase(MeteorologicalBase):
```

Comment:
Add a class that extends from `MeteorologicalInDBBase`. This represents properties to return to client. See `Coastline` in schemas/coastline.py for reference.